README.md

```diff
@@ -1,20 +1,157 @@
-[![Build Status](https://img.shields.io/azure-devops/build/responsibleai/fairlearn/6/dev?failed_label=bad&passed_label=good&label=GatedCheckin%3ADev)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=1&branchName=dev) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![pypi badge](https://img.shields.io/badge/pypi-0.2.0-blue)
+[![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# Fairness in Machine Learning - Mitigation algorithms
 
-A Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+Fairlearn
+=============================================================
+Fairlearn is a machine learning fairness toolkit that makes it easier for anyone involved in the development of Artificial Intelligence (AI) systems to monitor and understand the impact of unfairness in their machine learning lifecycle, and use state-of-the-art techniques to mitigate the observed unfairness. Fairlearn repository contains an SDK and Jupyter notebooks with examples to showcase its use.
```

Comment:
I think that the first sentence is a bit long, and could be split in to two (at least).

The second sentence should begin "The fairlearn repository"

Reply:
Also we said we want fairlearn to be with lowercase "f" at all times. If that seems odd at the start of the sentence, just make it `fairlearn` then it'll look like code

Reply:
I replaced the uppercase ones